app/pb_kits/playbook/pb_filter/_filter.jsx

```diff
@@ -0,0 +1,88 @@
+/* @flow */
+
+import React from 'react'
+import { Button, Caption, Card, CircleIconButton, Flex, Icon, PbReactPopover, SectionSeparator, Title, TitleCount } from '../'
+
+type FilterProps = {
+  className?: String,
+  data?: String,
+  id?: String,
+  children?: React.Node,
+  filters?: Array<Hash>,
+  sortMenu?: Array<Hash>
+}
+
+const Filter = ({
+  className = 'pb_filter_kit',
+  data,
+  id,
+  children,
+  sortMenu,
+  filters,
+}: FilterProps) => {
+  return (
+    <div
```

Comment:
If there's nothing other than the return statement, you could omit the curly braces
```
const Filter = ({
  className = 'pb_filter_kit',
  data,
  id,
  children,
  sortMenu,
  filters,
}: FilterProps) => (
  <div
```